etc/js/env-config.js

```diff
@@ -1,56 +1,29 @@
 export default {
     'http://localhost': {
         API_SERVER: 'https://localhost:3001',
+        WALLET_ADDRESS: 'http://localhost:3000',
+        EXPLORER: 'http://localhost:7001',
+        EXPLORER_VERSION: '0.1.0',
```

Comment:
Maybe we don't need this parameter anymore

Reply:
Removed the explorer_version param